client/src/components/ElementEditor/HoverBar.js

```diff
@@ -12,151 +51,29 @@ import { elementTypeType } from 'types/elementTypeType';
 class HoverBar extends Component {
   constructor(props) {
     super(props);
-
-    this.handleMouseEnter = this.handleMouseEnter.bind(this);
-    this.handleMouseLeave = this.handleMouseLeave.bind(this);
-    this.renderHoverBar = this.renderHoverBar.bind(this);
     this.toggle = this.toggle.bind(this);
-
     this.state = {
-      timeoutRef: null,
-      delayAreaActive: false,
-      instAreaActive: false,
       popoverOpen: false
     };
   }
 
-  /**
-   * Handle mouse entering the hover area between two elements. Calculates if mouse entered via
-   * 'delayed' or 'instantaneous' hover area and 'activates' these areas with a time out in the
-   * former case and instantaneously in the latter.
-   *
-   * @param event
-   */
-  handleMouseEnter(event) {
```

Comment:
I assume this is a breaking change, can you deprecate it instead?

Reply:
This method was meant to track the internal state of the HoverBar. I don't think this information was meant to be communicated back to any outside component. So if ES6 supported `private/protected/public` keywords, this method would have been `private`. In this context, I think it's safe to remove it altogether.

If I may jump on my favourite hobby horse, I would point out that TypeScript does support function access modifiers. Adopting TS would take the ambiguity out of this kind of front end work and allow us to make it clear what we consider to be part of our public API and what we consider to be internal logic.

Reply:
Alright - we still haven't formally introduced React into our semver commitments yet anyway, so kill it...

Reply:
Die Die Die `handleMouseEnter` method!!! :smiling_imp: